controller/backing_image_controller.go

```diff
@@ -212,6 +212,14 @@ func (bic *BackingImageController) syncBackingImage(key string) (err error) {
 			log.Info("Need to wait for all replicas stopping using this backing image before removing the finalizer")
 			return nil
 		}
+		log.Info("Try to clean up backing image data source tmp file before cleaning up backing image manager")
+		cleanUpTmpFileFinished, err := bic.cleanupBackingImageDataSourceTmpFile(backingImage)
+		if err != nil {
+			return err
+		}
+		if !cleanUpTmpFileFinished {
+			return nil
```

Comment:
This is intended, so suggest having an info log before returning nil, for examples, `Requesting to clean up Backing Image Data Source first before removing from Backing Image Manager`, you can add helpful parameters in the log.